src/variables/create.jl

```diff
@@ -0,0 +1,79 @@
+export compute_variables_indices
+
+function compute_variables_indices(connection, dataframes)
```

Comment:
Add "TODO: Allow changing table names to make unit tests possible."
Something like `compute_variables_indices(connection, assets_data="t_assets_data")`

Reply:
Done